packages/example/src/pages/guides/configuration.mdx

```diff
@@ -139,6 +140,35 @@ Global search is enabled by default. To disable it, set the `isSearchEnabled` op
 Under the hood, we use [Lunr](https://lunrjs.com/) to create our search index. If necessary, you tweak the search scoring algorithm and source nodes.
 To do so, provide your own [resolvers object](https://www.gatsbyjs.org/packages/gatsby-plugin-lunr/#getting-started) to the `lunrOptions` theme option.
 
+## Header nav
+
+The default header for the theme is the left nav sidebar. However, we have added a configuration option that allows you to switch from the default left nav to a header nav.
+
+We recommend using a header nav for sites that might not have many pages or tabs. If in the future, the content on your site evolves and you feel the left nav is better suited, you can always change it back.
+
+To enable the header nav, provide `header` as a string to the navigationStyle option. To switch back to the default left nav providee an empty string `` to the navigationStyle option.
```

Comment:
Looks great! Only comment is to update the empty string to be a code snippet bc it looks weird without it when you're reading the docs. 
```suggestion
To enable the header nav, provide `header` as a string to the navigationStyle option. To switch back to the default left nav providee an empty string `''` to the navigationStyle option.
```